polaris-pipeline/text-extractor/text-extractor.csproj

```diff
@@ -1,21 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
     <RootNamespace>text_extractor</RootNamespace>
     <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifier>linux-x64</RuntimeIdentifier>
```

Comment:
Is this property required? I have seen it commented out in other projects like the wire mock

Reply:
I found that it was required to include this to allow a the previous incarnation of the DDEI upgrade (that was subsequently abandoned) to start up when deployed to Azure. The windows version was also required for Windows apps such as PDF Generator, etc.

Reply:
Good spot! It is not, I have removed it.